pkg/utils/unhandled.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package controller
+package utils
```

Comment:
(nitpicking here) maybe an `errors` package? there'd definitely be a name collision with the stdlib's `errors` that we import in some places, but that could be handled similarly to `kerrors`

```
pkg/controller/runnable/reconciler_test.go
19:     "errors"
28:     kerrors "k8s.io/apimachinery/pkg/api/errors"
```

wdyt? could be a separate PR too if you agree